sync/src/relayer/transactions_process.rs

```diff
@@ -92,6 +94,18 @@ impl<'a> TransactionsProcess<'a> {
                     );
                     return None;
                 }
+                // skip txs with known bugs
+                if reject_known_bugs {
+                    if let Err(e) = IllTransactionChecker::new(&tx).check() {
```

Comment:
Does it be renamed back to the original name?

Reply:
Could you elaborate here? I'm not sure I get what this means

Reply:
I thought the name was KnownBugsChecker or alike when I reviewed last time

Reply:
Yes there's a suggestion to rename `KnownBugsChecker` to `IllTransactionChecker`. Do you mean to rename `reject_known_bugs` as well?

Reply:
My fault, I thought the comment 
 https://github.com/nervosnetwork/ckb/pull/1915#pullrequestreview-344408399 was asking to rename the class to KnownBugsChecker, instead it is the reverse.